libImgur/build.gradle

```diff
@@ -4,6 +4,6 @@ sourceCompatibility = 1.6
 targetCompatibility = 1.6
 
 dependencies {
-    compile "com.squareup.retrofit2:retrofit:${retrofitVersion}"
-    compile "io.reactivex:rxjava:${rxJavaVersion}"
-}
\ No newline at end of file
+    implementation "com.squareup.retrofit2:retrofit:${retrofitVersion}"
+    implementation "io.reactivex:rxjava:${rxJavaVersion}"
```

Comment:
I like how vars are used here to dynamically change the version of chosen libs/frameworks. Is this something that can be done with more of the libs in this project?

Reply:
Phil put this in. I surmise that it's particularly useful for the nested projects libImgur and libTba. I don't know why those are nested projects.

The build still gets into trouble when dependent libraries use different versions of these dependencies, not to mention outdated gradle commands like `compile` and task `<<`.

Reply:
It's particularly useful with `${supportLibVersion}`. It does add complexity and surprises, like when gradle complains that we're not using the same support library version everywhere (which must be due to an indirect dependency), so I'd want to extend it only to places where the same version number must appear in multiple places.

Reply:
Do the current build tools make retrofit obsolete?

Reply:
@1fish2 Nah, I think retrofit is still okay to include. That's how it is at my work's codebase at least!